File: app/src/main/java/com/casebeaumonde/fragments/Live_Events/response/FavLiveEventResponse.java
package com.casebeaumonde.fragments.Live_Events.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class FavLiveEventResponse {

@SerializedName("code")
@Expose
private String code;
@SerializedName("message")
@Expose
private String message;

public String getCode() {
return code;
}

public void setCode(String code) {
this.code = code;
}

public String getMessage() {
return message;
}

public void setMessage(String message) {
this.message = message;
}

}

File: app/src/main/java/com/casebeaumonde/fragments/Live_Events/response/FavLiveEventResponse.kt
package com.casebeaumonde.fragments.Live_Events.response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class FavLiveEventResponse {

    @SerializedName("code")
    @Expose
    private var code: String? = null

    @SerializedName("message")
    @Expose
    private var message: String? = null

    fun getCode(): String? {
        return code
    }

    fun setCode(code: String?) {
        this.code = code
    }

    fun getMessage(): String? {
        return message
    }

    fun setMessage(message: String?) {
        this.message = message
    }

}